src/components/Header/SearchHeader.tsx

```diff
@@ -41,37 +39,31 @@ const SearchHeader = ({
   // TODO (sgoldblatt): WHY are there so many wrappers?
   return (
     <SearchHeaderWrapper>
-      <Wrapper>
-        <Content>
-          <MenuBarWrapper>
-            <SelectorWrapper
-              onClick={() => {
-                setMobileMenuOpen(false);
-              }}
-              isNarrowMobile={isNarrowMobile}
-            >
-              <SearchAutocomplete
-                filterLimit={getFilterLimit(region)}
-                setHideMapToggle={setHideMapToggle}
-                locations={getFinalAutocompleteLocations(
-                  geolocationData,
-                  countyToZipMap,
-                )}
-              />
-            </SelectorWrapper>
-            {isMobile && (
-              <MapToggle
-                hideMapToggle={hideMapToggle}
-                onClick={() => toggleMobileMenu()}
-                isActive={mobileMenuOpen}
-              >
-                <>Counties&nbsp;&nbsp;</>
-                <MapIcon />
-              </MapToggle>
+      <MenuBarWrapper>
+        <SelectorWrapper
+          onClick={() => setMobileMenuOpen(false)}
+          $isNarrowMobile={isNarrowMobile}
+        >
+          <SearchAutocomplete
+            filterLimit={getFilterLimit(region)}
```

Comment:
unrelated, so feel free to punt, but i think we can remove this dynamic filter limit and just set it to a static number. now that we've taken out the location-specific ordering (ie. state's counties first, etc.), we don't need the location-specific filter limit. we always just rank alphabetized states first, and so currently each state is showing a different amount of states, which is strange